front/src/modules/metadata/components/ObjectShowPage.tsx

```diff
@@ -0,0 +1,159 @@
+import { IconBuildingSkyscraper } from '@/ui/display/icon';
+import { PageBody } from '@/ui/layout/page/PageBody';
+import { PageContainer } from '@/ui/layout/page/PageContainer';
+import { PageFavoriteButton } from '@/ui/layout/page/PageFavoriteButton';
+import { PageHeader } from '@/ui/layout/page/PageHeader';
+import { ShowPageContainer } from '@/ui/layout/page/ShowPageContainer';
+import { ShowPageAddButton } from '@/ui/layout/show-page/components/ShowPageAddButton';
+import { ShowPageLeftContainer } from '@/ui/layout/show-page/components/ShowPageLeftContainer';
+import { ShowPageRightContainer } from '@/ui/layout/show-page/components/ShowPageRightContainer';
+import { ShowPageSummaryCard } from '@/ui/layout/show-page/components/ShowPageSummaryCard';
+import { ShowPageRecoilScopeContext } from '@/ui/layout/states/ShowPageRecoilScopeContext';
+import { PageTitle } from '@/ui/utilities/page-title/PageTitle';
+import { RecoilScope } from '@/ui/utilities/recoil-scope/components/RecoilScope';
+
+import { useFindOneMetadataObject } from '../hooks/useFindOneMetadataObject';
+import { useFindOneObject } from '../hooks/useFindOneObject';
+import { useUpdateOneObject } from '../hooks/useUpdateOneObject';
+import { formatMetadataFieldAsColumnDefinition } from '../utils/formatMetadataFieldAsColumnDefinition';
+
+export const ObjectShowPage = () => {
+  const { objectNameSingular, objectId } = useParams<{
+    objectNameSingular: string;
+    objectId: string;
+  }>();
+
+  const { foundMetadataObject } = useFindOneMetadataObject({
+    objectNameSingular,
+  });
+
+  const [, setEntityFields] = useRecoilState(
+    entityFieldsFamilyState(objectId ?? ''),
+  );
+
+  const { object } = useFindOneObject({
+    objectId: objectId,
+    objectNameSingular,
+    onCompleted: (data) => {
+      setEntityFields(data);
+    },
+  });
+
+  const useUpdateOneObjectMutation: () => [(params: any) => any, any] = () => {
+    const { updateOneObject } = useUpdateOneObject({
+      objectNameSingular,
+    });
+
+    const updateEntity = ({
+      variables,
+    }: {
+      variables: {
+        where: { id: string };
+        data: {
+          [fieldName: string]: any;
+        };
+      };
+    }) => {
+      updateOneObject?.({
+        idToUpdate: variables.where.id,
+        input: variables.data,
+      });
+    };
+
+    return [updateEntity, { loading: false }];
+  };
+
+  const handleFavoriteButtonClick = async () => {
+    //
+  };
+
+  if (!object) return <></>;
+
+  return (
+    <PageContainer>
+      <PageTitle title={object.name || 'No Name'} />
+      <PageHeader
+        title={object.name ?? ''}
+        hasBackButton
+        Icon={IconBuildingSkyscraper}
+      >
+        <PageFavoriteButton
+          isFavorite={false}
+          onClick={handleFavoriteButtonClick}
+        />
+        <ShowPageAddButton
+          key="add"
+          entity={{
+            id: object.id,
+            type: ActivityTargetableEntityType.Company,
```

Comment:
Is this intended to have Company here for now?

Reply:
Yes, any other type isn't supported and we didn't support activities.